one.mixin.messenger.desktop.yml

```
@@ -0,0 +1,51 @@
+id: one.mixin.messenger.desktop
+runtime: org.gnome.Platform
+runtime-version: '42'
```

```suggestion
runtime-version: '43'
```

but do you really need the GNOME runtime?

---

The reason we choose GNOME runtime is that GNOME-42 provide libgtk2webkit ...

GNOME-43 is seems too new, we can not load libwebkit2gtk-4.0.so.37 in GNOME-43 runtime:

```
mixin_desktop: error while loading shared libraries: libwebkit2gtk-4.0.so.37: cannot open shared object file: No such file or directory
```